### ISSUE TITLE
Fails with cli-8.0.0

### ISSUE THREAD

**Issue by spinninghamster** [apache/cordova-plugin-statusbar#101]
Using PhoneGap with cli-8.0.0 it fails:

```
"plugman-151 install --platform android --project  --plugin cordova-plugin-statusbar --nohooks=.": Fetching plugin "cordova-plugin-statusbar" via npm
Installing "cordova-plugin-statusbar" at "2.4.2" for android
Failed to install 'cordova-plugin-statusbar': Error: ENOENT: no such file or directory, open '/AndroidManifest.xml'
    at Error (native)
    at Object.fs.openSync (fs.js:640:18)
    at Object.fs.readFileSync (fs.js:508:33)
    at Object.parseElementtreeSync (/cordova/node_modules/cordova-common/src/util/xml-helpers.js:180:27)
    at new AndroidManifest (/cordova/lib/AndroidManifest.js:29:20)
    at AndroidProject.getPackageName (/cordova/lib/AndroidProject.js:99:12)
    at Api.addPlugin (/cordova/Api.js:223:57)
    at handleInstall (/usr/local/lib/npm/lib/node_modules/pgb-plugman-151/node_modules/pgb-cordova-lib/src/plugman/install.js:594:10)
    at /usr/local/lib/npm/lib/node_modules/pgb-plugman-151/node_modules/pgb-cordova-lib/src/plugman/install.js:364:24
    at _fulfilled (/usr/local/lib/npm/lib/node_modules/pgb-plugman-151/node_modules/q/q.js:787:54)
ENOENT: no such file or directory, open '/AndroidManifest.xml'
```

**Comment by janpio** [comment]
Isn't this a Phonegap issue then?
This plugin is normally installed with `cordova plugin add cordova-plugin-statusbar`.

**Comment by jcesarmobile** [comment]
is it the only plugin you have? If not, list them all
what's your cordova-android version?

**Comment by pedoc** [comment]
+1,cordova-cli 8.0
cordova-android 7.1.1

can't install this plugin,but it's working for cordova android 6.x

**Comment by ThijmenDF** [comment]
Can confirm, this isn't a phonegap specific issue.

Running cordova-cli 8.1.2 & cordova-android 7.1.1
```
cordova plugin add cordova-plugin-statusbar --save -d
...
Failed to install 'cordova-plugin-statusbar': Error: ENOENT: no such file or directory, open 'C:\Users\<location>\platforms\android\AndroidManifest.xml'
    at Object.fs.openSync (fs.js:667:18)
    at Object.fs.readFileSync (fs.js:572:33)
    at Object.parseElementtreeSync (C:\Users\<location>\platforms\android\cordova\node_modules\cordova-common\src\util\xml-helpers.js:180:27)
    at new AndroidManifest (C:\Users\<location>\platforms\android\cordova\lib\AndroidManifest.js:29:20)
    at AndroidProject.getPackageName (C:\Users\frankrt\<location>\platforms\android\cordova\lib\AndroidProject.js:99:12)
    at Api.addPlugin (C:\Users\<location>\platforms\android\cordova\Api.js:223:57)
    at handleInstall (C:\Users\<location>\AppData\Roaming\npm\node_modules\cordova\node_modules\cordova-lib\src\plugman\install.js:594:10)
    at C:\Users\<location>\AppData\Roaming\npm\node_modules\cordova\node_modules\cordova-lib\src\plugman\install.js:357:28
    at _fulfilled (C:\Users\<location>\AppData\Roaming\npm\node_modules\cordova\node_modules\q\q.js:854:54)
    at self.promiseDispatch.done (C:\Users\<location>\AppData\Roaming\npm\node_modules\cordova\node_modules\q\q.js:883:30)
```

See this answer for more information about what's causing it:
https://stackoverflow.com/a/47927193

I'll try to find a way to fix this issue and post any progress here.

**Comment by ThijmenDF** [comment]
For me specifically, the main reason why it didn't work is because there was a 'res' folder in the Android root project folder. After moving it to `/app/src/main/res` it worked flawlessly again.

**Comment by jcesarmobile** [comment]
Yeah, this is a problem with other plugins adding the res/lib folder on the root on cordova-android 7.x.x. 
It's fixed in master and 7.1.x branch, but not released.
